kubernetes/curriculum.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 RUN_NAME="$1"
 VOLUME_NAME="$2"
+
+# not related to curriculum but we want some process to log this
```

Comment:
A nit, but I'd lean towards putting this in `kubernetes/train.sh` since we _always_ need train.sh to do anything, whereas we could in theory run without a curriculum.

Reply:
placed it in `shuffle-and-export` instead — `shuffle-and-export` doesn't request a GPU so it's more likely to start running sooner